dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/UserServiceBean.java

```diff
@@ -225,132 +207,88 @@ private AuthenticatedUser addAuthenticatedUserRoles(AuthenticatedUser authentica
                 "     AND e.containedexplicitgroups_id = c.id)" +
                 "SELECT distinct groupalias, useridentifier FROM group_user;";
 
-
-        //System.out.println("qstr: " + qstr);
-
         nativeQuery = em.createNativeQuery(qstr);
         List<Object[]> groupResults = nativeQuery.getResultList();
         if (groupResults == null) {
             return userRoleLookup;
         }
 
-        String groupIdentifiers = null;
+        Set<String> groupIdentifiers = new HashSet<>();
 
         for (Object[] group : groupResults) {
-            String alias = UserUtil.getStringOrNull(group[0]);
-            String user = UserUtil.getStringOrNull(group[1]);
-            if (alias != null) {
-
-                alias = "&explicit/" + alias;
+            String alias = getStringOrNull(group[0]);
+            String user = getStringOrNull(group[1]);
```

Comment:
I don't think that `getStringOrNull(group[1])` is any different than just doing `(String) group[1]`